### DISCUSSION TITLE
Question regarding DRUG_EXPOSURE drug dose calculation

### DISCUSSION THREAD

**Discussion by mkrile38** [chorus-ai/StandardsModule#95] [Structured EHR and ETL Questions] [answered]
My question pertains to calculating the quantity value in DRUG_EXPOSURE table, specifically when it comes to drugs administered via infusion. Our data source provides dosage in the form of micrograms per kilogram per minute. It does not include the pt weight used to calculate the dosage rate. In these situations, would it be acceptable to use this rate as quantity with unit concept id 9688 (ug/kg/min), or do we need to determine the kg value to calculate the total dosage of ingredient administered between begin and end time?

**Comment by jshoughtaling** [comment, marked as answer]
@mkrile38 - fantastic question, and one that is still the [subject of debate within the OHDSI community](https://forums.ohdsi.org/t/how-to-record-an-inpatient-drug-infusion/4711/16). The general convention for storing `quantity` of a liquid drug in the DRUG_EXPOSURE table relies on that drug's representation in the DRUG_STRENGTH table. I've copied the convention from the [CDM GitHub guide](https://ohdsi.github.io/CommonDataModel/cdm54.html#drug_exposure) below, with some relevant passages bolded:

>To find the dose form of a drug the RELATIONSHIP table can be used where the relationship_id is ‘Has dose form’. **If liquid, quantity stands for the total amount dispensed or ordered of ingredient in the units given by the drug_strength table.** If the unit from the source data does not align with the unit in the DRUG_STRENGTH table the quantity should be converted to the correct unit given in DRUG_STRENGTH. For clinical drugs with fixed dose forms (tablets etc.) the quantity is the number of units/tablets/capsules prescribed or dispensed (can be partial, but then only 1/2 or 1/3, not 0.01). **Clinical drugs with divisible dose forms (injections) the quantity is the amount of ingredient the patient got**. For example, if the injection is 2mg/mL but the patient got 80mL then quantity is reported as 160. Quantified clinical drugs with divisible dose forms (prefilled syringes), the quantity is the amount of ingredient similar to clinical drugs. Please see [how to calculate drug dose](https://ohdsi.github.io/CommonDataModel/drug_dose.html) for more information.

I gave a demo at an office hours a few months ago on using the DRUG_STRENGTH table in the context of DOSE_ERA calculations and it's possible I could share the recording with you if you're interested. The bottom line is: in order to convert your source data to OMOP and capture `quantity` appropriately, you would need to know (or infer from a prior measurement) the patient's weight used to calculate the dosage rate. It is likely there will be a weight measurement recorded during that same visit that you can use to extract the drug quantity. If that's not the case then I'm afraid you will need to leave that field blank and explore other options for storing infusion rate outside of DRUG_EXPOSURE (we can discuss further offline).
